### PR TITLE
Swift 6 Migration

### PR DESCRIPTION
**Pull request by OguzYuuksel**

### CODE REVIEW

**Review by GiuliaAriu — APPROVED** [review]

File: Sources/MergeL10n/Commands/PseudoToLanguages.swift [hunk 1]
```diff
@@ -102,7 +108,7 @@ struct PseudoToLanguages: ParsableCommand {
     private func run(folder: String, languages: [String], developmentLanguage: String) -> Reader<SimpleFileManager, Result<Void, LocalizedStringFileError>>{
         Reader { fileManager in
             LocalizedStringFile(basePath: folder, language: "zz")
-                .read(encoding: .utf16, requiresComments: true)
+                .read(encoding: .ascii, requiresComments: true)
```
Comment (GiuliaAriu) [inline Sources/MergeL10n/Commands/PseudoToLanguages.swift]: Why this change?

Comment (OguzYuuksel) [reply Sources/MergeL10n/Commands/PseudoToLanguages.swift]: Thanks for catching. It should not be, I will check it out.

Comment (OguzYuuksel) [reply Sources/MergeL10n/Commands/PseudoToLanguages.swift]:
```suggestion
                .read(encoding: .utf16, requiresComments: true)
```

File: Sources/MergeL10n/Commands/PseudoToLanguages.swift [hunk 2]
```diff
@@ -102,7 +108,7 @@ struct PseudoToLanguages: ParsableCommand {
     private func run(folder: String, languages: [String], developmentLanguage: String) -> Reader<SimpleFileManager, Result<Void, LocalizedStringFileError>>{
         Reader { fileManager in
             LocalizedStringFile(basePath: folder, language: "zz")
-                .read(encoding: .utf16, requiresComments: true)
+                .read(encoding: . utf16, requiresComments: true)
```
Comment (OguzYuuksel) [inline Sources/MergeL10n/Commands/PseudoToLanguages.swift]:
```suggestion
                .read(encoding: .utf16, requiresComments: true)
```
